### PR TITLE
torf-cli: init at 3.4.0

### PR DESCRIPTION
**Pull request by Synthetica9**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 121989` at 1cfa1724 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li>python38Packages.flatbencode</li>
    <li>python38Packages.torf</li>
    <li>python39Packages.flatbencode</li>
    <li>python39Packages.torf</li>
    <li>torf-cli</li>
  </ul>
</details>
<details>
  <summary>4 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/development/python-modules/flatbencode/default.nix:1:3:
```
  |
1 | { stdenv, lib, fetchFromGitHub, buildPythonPackage, hypothesis, pytestCheckHook
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/tools/misc/torf-cli/default.nix:1:3:
```
  |
1 | { stdenv, lib, python3Packages, fetchFromGitHub }:
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/development/python-modules/torf/default.nix:1:3:
```
  |
1 | { stdenv, lib, fetchFromGitHub, buildPythonPackage, pythonOlder, flatbencode
  |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/torf/default.nix:21:0:
```
   |
21 |     description =
   | ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/torf-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchFromGitHub {
+    owner = "rndusr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ad6swad8wpvxlivbrydhjrgjl82xd6pppjl88c6xfkzndydp15c";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ torf pyxdg ];
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/torf-cli/default.nix:16]: Why did you disable tests?

File: pkgs/development/python-modules/torf/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  propagatedBuildInputs = [ flatbencode ];
+  checkInputs = [ pytest ];
+
+  disabled = pythonOlder "3.6";
+
+  meta = with lib; {
+    description =
+      "Python module to create, parse and edit torrent files and magnet links";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/torf/default.nix:22]:
```suggestion
    description = "Python module to create, parse and edit torrent files and magnet links";
```

File: pkgs/development/python-modules/torf/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  src = fetchFromGitHub {
+    owner = "rndusr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0l9j45parz13xgsl40bf58zib0js6fs22fssx6x457q6drw7a2zg";
+  };
+
+  propagatedBuildInputs = [ flatbencode ];
+  checkInputs = [ pytest ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/torf/default.nix:16]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

File: pkgs/development/python-modules/torf/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  disabled = pythonOlder "3.6";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/torf/default.nix:18]:
```suggestion
```

File: pkgs/development/python-modules/torf/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, buildPythonPackage, pythonOlder, flatbencode
+, pytest }:
+
+buildPythonPackage rec {
+  pname = "torf";
+  version = "3.1.3";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/torf/default.nix:6]:
```suggestion
  version = "3.1.3";
  disabled = pythonOlder "3.6";
```

File: pkgs/development/python-modules/torf/default.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, buildPythonPackage, pythonOlder, flatbencode
+, pytest }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/torf/default.nix:2]:
```suggestion
, pytestCheckHook }:
```

File: pkgs/development/python-modules/flatbencode/default.nix [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    sha256 = "1vvyh9mwzjm8il41qm3gl1bssg48n7kl0p1h4nhaq81wa7ik1znp";
+  };
+
+  checkInputs = [ pytestCheckHook hypothesis ];
+
+  disabled = pythonOlder "3.4";
+
+  meta = with lib; {
+    description =
+      "Fast, safe and non-recursive implementation of Bittorrent bencoding for Python 3";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flatbencode/default.nix:21]:
```suggestion
  meta = with lib; {
    description = "Fast, safe and non-recursive implementation of Bittorrent bencoding for Python 3";
```

File: pkgs/development/python-modules/flatbencode/default.nix [hunk 8]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, buildPythonPackage, hypothesis, pytestCheckHook
+, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "flatbencode";
+  version = "0.2.1";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flatbencode/default.nix:6]:
```suggestion
  version = "0.2.1";
  disabled = pythonOlder "3.4";
```

**Comment by doronbehar** [comment]
@Synthetica9 please address the review comments, and fix the merge conflicts.

**Pull request closed without merging**